### ISSUE TITLE
Text display bug on some letters

### ISSUE THREAD

**Issue by sebastienroy** [Harbys/pico-ssd1306#2]
Hello,
Thank your for this simple but efficient driver.
It works fine for my usage, which is very basic.
I've however noticed that, for some letters with long legs, such as 'p', 'q', 'y' are not displayed very well. It's like if the bottom pixels where displayed at the top of the letter.

**Comment by Harbys** [comment]
Can you post your code example to reproduce the issue?

**Comment by sebastienroy** [comment]
Hello, I used exactly the same code as one of your example :+1: 

```
#include "pico/stdlib.h"
#include "pico-ssd1306/ssd1306.h"
#include "pico-ssd1306/textRenderer/TextRenderer.h"
#include "hardware/i2c.h"

// Use the namespace for convenience
using namespace pico_ssd1306;

int main() {
    // Init i2c0 controller
    i2c_init(i2c0, 1000000);
    // Set up pins 12 and 13
    gpio_set_function(20, GPIO_FUNC_I2C);
    gpio_set_function(21, GPIO_FUNC_I2C);
    gpio_pull_up(20);
    gpio_pull_up(21);

    // If you don't do anything before initializing a display pi pico is too fast and starts sending
    // commands before the screen controller had time to set itself up, so we add an artificial delay for
    // ssd1306 to set itself up
    sleep_ms(250);

    // Create a new display object at address 0x3D and size of 128x64
    SSD1306 display = SSD1306(i2c0, 0x3C, Size::W128xH64);

    // Here we rotate the display by 180 degrees, so that it's not upside down from my perspective
    // If your screen is upside down try setting it to 1 or 0
    display.setOrientation(0);

    // Draw 8x8
    drawText(&display, font_8x8, "Ready pqr", 0, 0);

    // Draw 12x16
    drawText(&display, font_12x16, "Ready pqr", 0, 16);


    // Send buffer to the display
    display.sendBuffer();

}

```

**Comment by sebastienroy** [comment]
![ssd1306](https://user-images.githubusercontent.com/28447454/154346511-17609ce7-dcd8-41c6-9fa0-ba04f1065f66.jpg)

Look at the pixels at the top of the y, q and q letters

**Comment by Harbys** [comment]
It looks like a bug in font shifting in the library. I will look into it tomorrow

**Comment by Harbys** [comment]
It seems I made a simple mistake in the script that generated font files, it shifted wrong between the lines. Fixed in [748a09e](https://github.com/Harbys/pico-ssd1306/commit/748a09e87d9c65c265b7c8a5e8ea9c16a8572de6).

After building your example (and some different ones) with the updated library no bugs appeared
![oled](https://user-images.githubusercontent.com/44087388/154584328-aea37ba8-f3d6-4f07-a581-1aa617b0af34.jpg)

If you find any more issues please feel free to open a new issue

**Comment by sebastienroy** [comment]
Thank you very much for your fix. It's fine now.
